app/scenes/right_menu_drawer/right_menu_drawer.js

```diff
@@ -49,14 +50,43 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
 
 export default class RightMenuDrawer extends React.Component {
     static propTypes = {
+        errors: React.PropTypes.array.isRequired,
+        currentUserId: React.PropTypes.string.isRequired,
+        currentTeamId: React.PropTypes.string.isRequired,
         actions: React.PropTypes.shape({
             goToModalAccountSettings: React.PropTypes.func.isRequired,
             goToModalSelectTeam: React.PropTypes.func.isRequired,
+            clearErrors: React.PropTypes.func.isRequired,
             logout: React.PropTypes.func.isRequired
         }).isRequired,
         theme: React.PropTypes.object
     }
 
+    openErrorEmail = () => {
+        const recipient = 'feedback@mattermost.com';
+        const subject = 'Problem with Mattermost React Native app';
+        Linking.openURL(
+            `mailto:${recipient}?subject=${subject}&body=${this.errorEmailBody()}`
```

Comment:
URI encoding the mailto url, in part or whole, screws up the email. Either the entire string is treated as the recipient/subject or the body is one big %-encoded blob.

And I've tried opening the email (w/o uri encoding) with all sorts of errors, even ones with html responses inside of the error objects, and I haven't had a problem.
	
We suspect that the `Linking` library is performing the encoding automatically.